remote.go

```diff
@@ -1095,8 +1095,96 @@ func (wd *remoteWD) KeyUp(keys string) error {
 	return wd.keyAction("keyUp", keys)
 }
 
-// TODO(minusnine): Implement PerformActions and ReleaseActions, for more
-// direct access to the W3C specification.
+func (wd *remoteWD) KeyPauseAction(duration time.Duration) KeyAction {
+	return KeyAction{
+		"type":     "pause",
+		"duration": uint(duration / time.Millisecond),
+	}
+}
+
+func (wd *remoteWD) KeyUpAction(key string) KeyAction {
+	return KeyAction{
+		"type":  "keyUp",
+		"value": key,
+	}
+}
+
+func (wd *remoteWD) KeyDownAction(key string) KeyAction {
+	return KeyAction{
+		"type":  "keyDown",
+		"value": key,
+	}
+}
+
+func (wd *remoteWD) PointerPauseAction(duration time.Duration) PointerAction {
+	return PointerAction{
+		"type":     "pause",
+		"duration": uint(duration / time.Millisecond),
+	}
+}
+
+func (wd *remoteWD) PointerMoveAction(duration time.Duration, offset Point, origin PointerMoveOrigin) PointerAction {
+	return PointerAction{
+		"type":     "pointerMove",
+		"duration": uint(duration / time.Millisecond),
+		"origin":   origin,
+		"x":        offset.X,
+		"y":        offset.Y,
+	}
+}
+
+func (wd *remoteWD) PointerUpAction(button MouseButton) PointerAction {
+	return PointerAction{
+		"type":   "pointerUp",
+		"button": button,
+	}
+}
+
+func (wd *remoteWD) PointerDownAction(button MouseButton) PointerAction {
+	return PointerAction{
+		"type":   "pointerDown",
+		"button": button,
+	}
+}
+
+func (wd *remoteWD) StoreKeyActions(inputId string, actions ...KeyAction) {
```

Comment:
Initialisms in Go have a consistent case. Here: `inputID`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms